optimade/models/links.py

```diff
@@ -61,27 +59,28 @@ class LinksResourceAttributes(Attributes):
 
     link_type: LinkType = Field(
         ...,
-        description="The link type of the represented resource in relation to this implementation. MUST be one of these values: 'child', 'root', 'external', 'providers'.",
+        description="""The link type of the represented resource in relation to this implementation.
```

Comment:
This should either be changed or contributed back to the spec.

Reply:
The spec just says "the type of the linked relation", I would rather we just go with that for the sake of this PR.